kustomize/commands/edit/add/addconfigmap.go

```diff
@@ -11,6 +11,7 @@ import (
 	"sigs.k8s.io/kustomize/api/resource"
 	"sigs.k8s.io/kustomize/api/types"
 	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/util"
 	"sigs.k8s.io/kustomize/kyaml/filesys"
 )
```

Comment:
nit: Saw the  name of the file here changed, is that needed? It's already under the `add` subdirectory, so not sure that we need to re-specify `add` in the file name

Reply:
Not needed, but I think it'd be nice to have for consistency with the other files in the same folder.

Reply:
I undid this in the new #5409 PR.

Reply:
Oh I see, that's a good point - didn't realize that the other files are doing it this way as well. In that case I agree with you that it is good to have the consistency; feel free to add the change back if you'd like.